app/controllers/users.js

```diff
@@ -0,0 +1,28 @@
+const { body, validationResult } = require('express-validator');
+
+const { createUser } = require('../services/users');
+const { fieldValidationError } = require('../errors');
+const { creationParamsMapper } = require('../mappers/users');
+const { showSerializer } = require('../serializers/users');
+
+exports.create = [
+  body('user.first_name', 'first_name must be present')
+    .not()
+    .isEmpty(),
+  body('user.last_name', 'last_name must be present')
+    .not()
+    .isEmpty(),
+  body('user.password', 'password must be present')
+    .not()
+    .isEmpty(),
+  body('user.email', 'email must have email format').isEmail(),
```

Comment:
Remember that we do requests validation on middlewares.
Also, we have to check that the fields have the rigth type.